applications: serial_lte_modem: Init error msg and and ignore cloud fail (#22214)
nrfconnect/sdk-nrf: applications/serial_lte_modem/doc/slm_description.rst

#. Observe that the development kit sends a ``Ready\r\n`` message on UART.
#. Send AT commands and observe the responses from the development kit.

See :ref:`slm_testing` for typical test cases.

.. note::

>    If initialization of SLM fails, the application will send ``INIT ERROR\r\n`` message on UART if it has been capable of enabling the UART.
>    More information about the error can be found in the logs.

Reviewer: ```suggestion
   If the initialization of SLM fails, the application sends an ``INIT ERROR\r\n`` message on UART if it has managed to enable UART.
   See the logs for more information about the error.
```
